examples/leds_shell/Makefile

```diff
@@ -18,5 +18,7 @@ QUIET ?= 1
 # Modules to include:
 USEMODULE += shell
 USEMODULE += periph_gpio
+# Prevents native program from segfault when gpio set/clear are used
+USEMODULE += periph_gpio_mock
```

Comment:
You want to limit that to the `native` boards.
But even `native` can use real GPIOs when you specify the `--gpio` parameter.

Reply:
I know that there is a possiblity to use real GPIO of the machine - but the idea of the `leds_shell` is that this is a simple program for Newbies - that probably do not use GPIO (at first) but due to curiosity could use `gpio set` or `gpio clear` which should not end with crash.

Reply:
so why not 
```suggestion
ifneq (,$(filter native native64,$(BOARD)))
  # Prevents native program from segfault when no gpiochip is specified
  USEMODULE += periph_gpio_mock
endif
```

Reply:
Yes, conditional addition of `perihp_gpio_mock` for `native` is a good idea.

The version with `native%` did not work - I leave `native` - and now everything works perfectly!

Reply:
`ifneq (,$(filter native native64,$(BOARD)))` should do then